src/constants.ts

```diff
@@ -1 +1 @@
-export const ITEMS_PER_PAGE = 15
+export const ITEMS_PER_PAGE = 16
```

Comment:
It's the number of initial products on the PLP. It's supposed to be 24 but [LH is complaining about the excessive DOM size](https://github.com/vtex-sites/base.store/runs/4875869736#:~:text=Avoid%20an%20excessive%20DOM%20size). Even dropping to 16 items seems to cause the issue and I haven't tried going lower yet to see if it fixes it. To be instigated.